CHANGELOG.md

```diff
@@ -8,9 +8,32 @@
 * For the time being, MapboxCoreNavigation.framework depends on a build of MapboxNavigationNative.framework that is only available to authorized beta testers. Please contact your Mapbox sales representative or [support team](https://support.mapbox.com/) to learn more about the beta testing program. ([#2412](https://github.com/mapbox/mapbox-navigation-ios/pull/2412))
 * MapboxNavigation depends on Mapbox Maps SDK for iOS v6.0.0. Before CocoaPods or Carthage can download Mapbox.framework, you need to create a special-purpose access token. See [the updated installation instructions in the readme](./README.md#installing-the-latest-prerelease) for more details. ([#2437](https://github.com/mapbox/mapbox-navigation-ios/pull/2437)) 
 * Xcode 11.4.1 or above is now required for building this SDK from source. ([#2417](https://github.com/mapbox/mapbox-navigation-ios/pull/2417))
+
+### Dependencies updates
+
+* Updated `MapboxNavigationNative` version to `xx.yy.zz` ([]())
```

Comment:
I think we've updated `Mapbox.framework` to `6.0.0` as well. It's probably also worth mentioning that `MapboxCommon.framework` is now weakly linked.

Reply:
The latest changelog on release-v1.0-pre-registry addresses the point about MapboxCommon.framework being added as a dynamic dependency.